### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by yaaaaa** [Zeruel87/Cameo-mod#32]
Openra can use more pixels in grid size, to make YR units visible-enoth, and it make usable NEW units or terrain from RA2/YR or Dune 2000 (or another game) without zip/brake graphic in future.

**Comment by patrickwieth** [comment]
Yeah we had a discussion about this some time ago in our discord. It is a lot of work to migrate all the existing stuff to a larger base grid. If you want to do it, come join us at our Discord and I guess we would be ok to migrate to a higher resolution base grid.
